packages/schematics/angular/application/schema.json

```diff
@@ -3,14 +3,15 @@
   "id": "SchematicsAngularApp",
   "title": "Angular Application Options Schema",
   "type": "object",
+  "description": "Generates a new basic app definition in the \"projects\" subfolder of the workspace.",
   "properties": {
     "projectRoot": {
-      "description": "The root directory of the new application.",
+      "description": "The root folder of the new app.",
```

Comment:
just a small nit, but in general we use `directory` instead of `folder` as terminology

Reply:
Jury is out on this one. I tend to prefer folder. 
@jenniferfell 
The Author's Style Guide does not address the issue -- and in fact, that page uses "folder" 19 times and "directory" 12 times - sometimes in the same sentence.